tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/FileTablespace.java

```diff
@@ -477,12 +503,17 @@ public long getMinSplitSize() {
     return diskIds;
   }
 
+  public List<Fragment> getSplits(String tableName, TableMeta meta, Schema schema, Path... inputs)
+    throws IOException {
+    return getSplits(tableName, meta, schema, null, inputs);
+  }
+
   /**
    * Generate the list of files and make them into FileSplits.
    *
    * @throws IOException
    */
-  public List<Fragment> getSplits(String tableName, TableMeta meta, Schema schema, Path... inputs)
+  public List<Fragment> getSplits(String tableName, TableMeta meta, Schema schema, String[] partitions, Path... inputs)
```

Comment:
I'm going to give a suggestion. We need to keep each method for one purpose if possible. That approach would keep logic simpler.